### PR TITLE
Address Daniel Migault, Russ Housley and Rich Salz's comments.

### PR DESCRIPTION
**Pull request by grittygrease**

### CODE REVIEW

File: draft-ietf-tls-subcerts.md [hunk 1]
```diff
@@ -133,7 +133,9 @@ an outage at a CA will negatively affect the uptime of the service.
 To reduce the dependency on external CAs, this document proposes a limited delegation
 mechanism that allows a TLS peer to issue its own credentials within
 the scope of a certificate issued by an external CA.  These credentials only enable the
-recipient of the delegation to speak for names that the CA has authorized.
+recipient of the delegation to speak for names that the CA has authorized.  Furthermore,
+this mechanism allows the server to use modern signature algorithms such as
+ed25519 even if their CA does not support them.
```
Comment (chris-wood) [inline draft-ietf-tls-subcerts.md:138]:
```suggestion
Ed25519 {{?RFC8032}} even if their CA does not support them.
```

File: draft-ietf-tls-subcerts.md [hunk 2]
```diff
@@ -342,7 +344,9 @@ has the following structure:
 valid_time:
 
 : Time in seconds relative to the beginning of the delegation certificate's
-  notBefore value after which the delegated credential is no longer valid. This MUST NOT exceed 7 days.
+  notBefore value after which the delegated credential is no longer valid.
+  Credentials with valid_times exceeding 7 days will be rejected (as described
+  in {{client-and-server-behavior}}).
```
Comment (chris-wood) [inline draft-ietf-tls-subcerts.md:349]:
```suggestion
  Endpoints will reject delegate credentials with valid_times exceeding 7 days (as described
  in {{client-and-server-behavior}}).
```

File: draft-ietf-tls-subcerts.md [hunk 3]
```diff
@@ -496,6 +500,7 @@ peer's expected identity.  It also takes the following steps:
 5. Use the public key in the peer's end-entity certificate to verify the
    signature of the credential using the algorithm indicated by
    DelegatedCredential.algorithm.
+6. Validate that DelegatedCredential.cred.valid_time is no more than 7 days.
```
Comment (chris-wood) [inline draft-ietf-tls-subcerts.md:503]: Should this be part of step (1), perhaps?

File: draft-ietf-tls-subcerts.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-tls-subcerts.md:349]: Also, the the client-and-server-behavior anchor defined? I checked the source and don't see `{#client-and-server-behavior}` anywhere.

Comment (grittygrease) [reply draft-ietf-tls-subcerts.md:349]: It works automatically. Surprise to me too.

File: draft-ietf-tls-subcerts.md (hunk 3 above)
Comment (grittygrease) [reply draft-ietf-tls-subcerts.md:503]: Good idea!

File: draft-ietf-tls-subcerts.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-tls-subcerts.md]: Hah, cool!

**Pull request merged**